docs/models/advanced/modeldata.md

```diff
@@ -0,0 +1,61 @@
+Model Data
+==========
+
+Model data is the successor of "extended blockstates" allowing modders to pass in extra data to be used in [`IBakedModel::getQuads`][getquads] when rendering blocks. This is particularly useful in complex models such as for camouflage blocks which use other block's models as an example. This is not to be confused with a [`TileEntityRenderer`][ter] which is used for dynamic rendering such as for animated blocks like a chest or any blocks which cannot be defined by a list of `BakedQuad`s.
+
+IBakedModels vs TERs
+------------------
+
+An [`IBakedModel`][bakedmodel] simply provides a list of `BakedQuad`s to the `BlockModelRenderer` which are then drawn to the screen. There are other properties available which define aspects which effect rendering such as whether the model supports ambient occlusion (a rendering effect). Most [`IBakedModel`][bakedmodel]s come from when a model file is loaded and then "baked" so that the model does not need to be loaded again.
+
+A [`TileEntityRenderer`][ter] or [`TER`][ter] (previously [`TileEntitySpecialRenderer`][ter] or [`TESR`][ter]) is used to render blocks in a way that cannot be represented with a static baked model (JSON, OBJ, B3D, others).
```

Comment:
This could be misleading, since you are using "or" and "previously" but always linking to the same thing. Maybe have only the first be a link?

Reply:
The statement about TERs was copied directly from the TER page as I felt that summed it up better than I could word it but I agree with the linking and will change that, thank you :)